hoodie-common/src/main/java/com/uber/hoodie/common/table/log/HoodieLogFileReader.java

```diff
@@ -210,15 +213,41 @@ private HoodieLogBlock readBlock() throws IOException {
   private HoodieLogBlock createCorruptBlock() throws IOException {
     log.info("Log " + logFile + " has a corrupted block at " + inputStream.getPos());
     long currentPos = inputStream.getPos();
-    long nextBlockOffset = scanForNextAvailableBlockOffset();
+    byte[] corruptedBytes;
+    int corruptedBlockSize = 0;
+    boolean done = true;
+    // read upto logblocksize to find next magic header
+    do {
+      corruptedBytes = new byte[logBlockSize];
```

Comment:
instead of handling this manually, should we just wrap this into a BufferedInputStream? 

https://docs.oracle.com/javase/7/docs/api/java/io/BufferedInputStream.html

Reply:
I see this is a DataInputStream.. still , is there some standard class that can give you the buffering for free